### DISCUSSION TITLE
Working with indents

### DISCUSSION THREAD

**Discussion by pjmattingly** [dendronhq/dendron#3947] [Q&A] [answered]
Hi,

I'm getting started with Dendron, and I wanted to see if it would be possible to use indentation with Dendron?

For example, I prefer to have my notes indented like:

```
Title
        sub-task
                sub-sub-task
```

Then I can use code folding to keep my documents compact. I get visually overwhelmed with lot's of text on the screen. And so Dendron's approach of using markdown headings is unappealing to me as it's very visually busy:

```
# Title
## sub-task
### sub-sub-task
```

A reasonable compromise seems to be to do something in-between:

```
# Title
        ## sub-task
                ### sub-sub-task
```

Then to turn this into valid markdown, whitespace just needs to be trimmed from the front. Is there a way I can do this?

Thanks.

**Comment by dysbulic** [comment, marked as answer]
Can you use nested lists to get the effect you want? *(Indents are two spaces.)*

* Title
  *  Sub-task
    * Sub-sub-task

**Reply by pjmattingly** [reply]
That seems reasonable.

Would it be possible to make a configuration change to change the indent size to 4? I find that easier to avoid the "visual overwhelming" issues I mentioned.

Also it would be very helpful if I could assign the "tab" key to insert `-` to create a (sub)list. That might be out of scope for Dendron. Do you have any pointers on how I might customize Code to make that change?

Thank you.

**Reply by dysbulic** [reply]
> Would it be possible to make a configuration change to change the indent size to 4?

Search me. 😸 I'm just getting into modifying Dendron & don't know how to do much of anything yet.

**Reply by pjmattingly** [reply]
Fair enough, thank you for your help. =)
